README.md

```diff
@@ -436,6 +436,14 @@ cp bootstrap/ansible/main.copyme main.yml
 ansible-playbook bootstrap/ansible/playbook.yml
 ```
 
+### Dynamic Settings
+
+Some configuration may need to be updated without a server restart (e.g., links
+to external resources). Such configuration is managed by `django-constance` and
+stored in Redis. To update these, navigate to `/admin/constance/config/`, and
```

Comment:
It took me a while to realize that `/admin/constance/config/` was a path on the website and not the VM filesystem, so I think it would be a good idea to make that clearer here, as I had figured the variables would be stored in config files in a directory by that name.

Reply:
Thanks, I just clarified this.